mgmtsystem_manual/checklog-odoo.cfg

```diff
@@ -0,0 +1,3 @@
+[checklog-odoo]
+ignore=
+WARNING.0 failed, 0 error\(s\).
```

Comment:
I'm curious, I'm not aware of this meta file being needed. Was it added by mistake?

Reply:
`checklog-odoo.cfg` is added to avoid test errors